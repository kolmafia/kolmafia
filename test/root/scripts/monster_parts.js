const { print } = require("kolmafia");

print("$monster[none] has no parts: " + (Monster.none.parts.length === 0 ? "true" : "false"));

const actual = Monster.get("Crimbuccaneer retiree").parts;
const expected = ["armpeg", "headpeg", "legpeg", "torsopeg"];

print("Reports all parts: " + (actual.length === expected.length && expected.every(p => actual.includes(p)) ? "true" : "false"));