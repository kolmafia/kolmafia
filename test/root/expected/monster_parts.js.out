$monster[none] has no parts: true
Reports all parts: true